src/screens/App.css

```
@@ -57,3 +62,25 @@
+
+#droplet-canvas {
+  background: linear-gradient(to bottom, #0f2027, #203a43, #2c5364);
+  box-shadow: 0 20px 50px rgba(0, 0, 0, 0.5);
+}
+
+#abcl-minigame-overlay {
+  z-index: 100;
+}
+
+.abcl-number-widget {
+}
```

Stylelint (17.14.0)

[error] Empty block (block-no-empty)

(block-no-empty)